lib/evmone/instruction_traits.hpp

```diff
@@ -1,12 +1,20 @@
 // evmone: Fast Ethereum Virtual Machine implementation
 // Copyright 2020 The evmone Authors.
 // SPDX-License-Identifier: Apache-2.0
+#pragma once
 
 #include <evmc/instructions.h>
 #include <array>
 
 namespace evmone::instr
 {
+/// EIP-2929 constants (https://eips.ethereum.org/EIPS/eip-2929).
+inline constexpr auto cold_sload_cost = 2100;
+inline constexpr auto cold_account_access_cost = 2600;
+inline constexpr auto warm_storage_read_cost = 100;
+inline constexpr auto additional_cold_account_access_cost =
```

Comment:
Maybe introduce also `additional_sload_cost` and use it in `sload` for consistency.

Reply:
I used the name, but only locally where it is used. Also explained the global definition.